extensions.json

```
@@ -388,6 +388,12 @@
       "download": "https://github.com/eamodio/vscode-gitlens/releases/download/v11.2.1/gitlens-11.2.1.vsix",
       "version": "11.2.1"
     },
+    {
+      "id": "eclipse-cdt.cdt-gdb-vscode",
+      "repository": "https://github.com/eclipse-cdt/cdt-gdb-vscode",
+      "version": "0.0.91",
+      "checkout": "master"
```

Hmm, the script didn't find a better Git ref than `master` for this version. Maybe let's skip `"version"` and just always publish from the default branch:

```suggestion
      "repository": "https://github.com/eclipse-cdt/cdt-gdb-vscode"
```